src/main/java/seedu/address/model/InternshipDiary.java

```diff
@@ -105,7 +106,7 @@ public String toString() {
 
     @Override
     public ObservableList<InternshipApplication> getInternshipList() {
-        return internships.asUnmodifiableObservableList();
+        return internships.asUnmodifiableObservableList().sorted(new ApplicationDateThenInterviewDateComparator());
```

Comment:
Why is internship list being sorted by default? This is causing test to fail.

Reply:
Perhaps I heard wrongly the last meeting, I will remove it.